WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -54,6 +58,26 @@
   </div>
 </div>
 
+<% if @display == 'admin' %>
+  <div class="form-group">
+    <%= label_tag(:status) %>
+    <div id="status" class="btn-group" data-toggle="buttons">
+      <label id="all" class="btn btn-info">
+        <%= radio_button_tag "status", "all" %>
+        <span class="caption">All</span>
+      </label>
+      <label id="warning" class="btn btn-warning">
+        <%= radio_button_tag "status", "warning" %>
+        <span class="caption">Warning</span>
+      </label>
+      <label id="danger" class="btn btn-danger">
+        <%= radio_button_tag "status", "danger" %>
+        <span class="caption">Danger</span>
+      </label>
+    </div>
+  </div>
+<% end %>
```

Comment:
Indeed, @jfly and I found this yesterday. Not sure what would be a reasonable solution, though.